test/e2e/specs/updates/scaleoutin/scaleoutin.go

```diff
@@ -0,0 +1,216 @@
+package scaleoutin
```

Comment:
can this go directly in test/e2e/specs/scaleupdown.go?

Reply:
done